tests/Giraffe.Tests/Giraffe.Tests.fsproj

```diff
@@ -18,6 +18,9 @@
     <PackageReference Include="NSubstitute" Version="3.1.*" />
     <PackageReference Include="FsCheck" Version="3.0.0-*" />
     <PackageReference Include="FsCheck.Xunit" Version="3.0.0-*" />
+    <DotNetCliToolReference Include="dotnet-xunit" Version="2.3.1" />
+    <DotNetCliToolReference Include="Microsoft.DotNet.Watcher.Tools" Version="2.0.0" />
```

Comment:
What is the Watcher used for?

Reply:
I usually put this on all applications. This allows me.to write `dotnet watch command` and have it rebuild and rerun/retest whenever I save a file. 

Can remove.it needed.